pandas/_libs/tslibs/offsets.pyx

```diff
@@ -3598,9 +3598,10 @@ cpdef to_offset(freq):
                 if not stride:
                     stride = 1
 
-                from .resolution import Resolution  # TODO: avoid runtime import
+                # TODO: avoid runtime import
+                from .resolution import Resolution, reso_str_bump_map
```

Comment:
can you do an absolute import

Reply:
sure, follow-up?  this is a blocker for stuff im excited about

Reply:
(and actually #34486 will get rid of this import)